repos/system_upgrade/el7toel8/actors/networkmanagerreadconfig/libraries/library.py

```diff
@@ -0,0 +1,45 @@
+from six.moves.configparser import ConfigParser, ParsingError
+
+from leapp.libraries.stdlib import CalledProcessError, run, api
+
+
+def read_nm_config(file_path=None):
+    if file_path:
+        try:
+            with open(file_path, 'r') as f:
+                r = f.read()
+                return r
+        except IOError:
+            return None
```

Comment:
I would suggest adding warning log entry similar to the one when running `NetworkManager --print-config` fails.